.github/workflows/pr-analysis-gradle.yml

```diff
@@ -35,10 +38,29 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Checkout for determining submodules
+        uses: actions/checkout@v3
+
+      - name: Determine Submodules
+        id: check-submodules
+        run: |
+          if [ -f .gitmodules ]; then
+            echo "has_submodules=recursive" >> $GITHUB_OUTPUT
```

Comment:
Consider renaming the output variable, the name suggests a boolean but the value is either `false` or `recursive`

Reply:
If I use "submodule_type" or "submodule_exists" both of them are the same as "has_submodules". Would you please suggest a name?

Reply:
I found one :smile: "submodules_status"

Reply:
Thanks. I will go for "submodule_checkout_strategy"